<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleSubWindow</class>
 <widget class="QWidget" name="ConsoleSubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="cmdButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
